pkg/shell/hosts.jsx

```diff
@@ -184,13 +184,14 @@ export class CockpitHosts extends React.Component {
                                     <HostLine user={user} host={label} />
                                 </span>
                             </span>
-                            {this.state.opened && <CaretUpIcon className="pf-v5-c-select__toggle-arrow mobile_v pf-v5-c-icon pf-m-lg" aria-hidden="true" />}
-                            {!this.state.opened && <CaretDownIcon className="pf-v5-c-select__toggle-arrow mobile_v pf-v5-c-icon pf-m-lg" aria-hidden="true" />}
+                            
+                            <CaretUpIcon className={`pf-v5-c-select__toggle-arrow mobile_v  ${this.state.opened ? 'clicked' : ''
+                            } pf-v5-c-icon pf-m-lg`} aria-hidden="true" />
```

Comment:
This is indented badly, the `}` shouldn't be on the same column as the initial tag, but indented more. It usually looks best to put one attribute per line, and aligning them.